pmm-app/tests/helper/grafana_helper.js

```diff
@@ -0,0 +1,29 @@
+const { Helper } = codeceptjs;
+
+class Grafana extends Helper {
+  // before/after hooks
+  /**
+   * @protected
+   */
+  _before() {
+    // remove if not used
+  }
+
+  /**
+   * @protected
+   */
+  _after() {
+    // remove if not used
+  }
+
+  // add custom methods here
+  // If you need to access other helpers
+  // use: this.helpers['helperName']
+
+  async Authorize() {
+    const { browserContext } = this.helpers.Playwright;
+    await browserContext.setExtraHTTPHeaders({ Authorization: 'Basic YWRtaW46YWRtaW4=' });
```

Comment:
Although this is for running the tests, but not really the best practice to hard-code the login/password. Can we pass via `.env` file?